[Bug]: 4.18: inconsistent failuer of tests/functional/upgrade/test_configuration.py::test_update_strategy_config_change with AssertionError: assert '2' == '1'

### Description

Inconsistent failuer of test_update_strategy_config_change[csi-cephfsplugin-2-2] with AssertionError: assert '2' == '1' .


### Steps to Reproduce
1. https://reportportal-ocs4.apps.dno.ocp-hub.prod.psi.redhat.com/ui/#ocs/launches/795/42373/1949891/1950085/log
2.
3.

### Actual Behavior
daemonset = 'csi-cephfsplugin', value_to_set = 2, expected_value = 2
rook_operator_configmap_cleanup = None


@pytest.mark.parametrize(
    argnames=["daemonset", "value_to_set", "expected_value"],
    argvalues=[
        pytest.param(
            "csi-rbdplugin", 2, 2, marks=[tier1, pytest.mark.polarion_id("OCS-6276")]
        ),
        pytest.param(
            "csi-cephfsplugin", 2, 2, marks=[tier1, pytest.mark.polarion_id("OCS-6277")]
        ),
    ],
)
@brown_squad
def test_update_strategy_config_change(
    daemonset, value_to_set, expected_value, rook_operator_configmap_cleanup
):
    """
    Test that tested value added to configmap rook-ceph-operator-config is
    reflected in respective daemonset.
    """
    if daemonset == "csi-rbdplugin":
        parameter_name = "CSI_RBD_PLUGIN_UPDATE_STRATEGY_MAX_UNAVAILABLE"
    elif daemonset == "csi-cephfsplugin":
        parameter_name = "CSI_CEPHFS_PLUGIN_UPDATE_STRATEGY_MAX_UNAVAILABLE"

    config_map_patch = f'\'{{"data": {{"{parameter_name}": "{value_to_set}"}}}}\''
    exec_cmd(
        f"oc patch configmap -n {config.ENV_DATA['cluster_namespace']} "
        f"{constants.ROOK_OPERATOR_CONFIGMAP} -p {config_map_patch}"
    )
    ds_obj = DaemonSet(
        resource_name=daemonset, namespace=config.ENV_DATA["cluster_namespace"]
    )
    results = ds_obj.get_update_strategy()

  assert str(expected_value) == str(results["rollingUpdate"]["maxUnavailable"])
E       AssertionError: assert '2' == '1'
E         - 1
E         + 2



tests/functional/upgrade/test_configuration.py:148: AssertionError
### Expected Behavior
Test must pass
### Impact (likelihood of reproduction, impact on the cluster and on other tests, etc.)


### Screenshots (if applicable)

### Environment
- Test Suite(s):
- Platform(s):
- Version(s):
- OS:

### Additional Context

Looking at below 2 logs this seems to be an temparary timimg issue. 
before 
time.sleep(10)  # wait a few second
    results = ds_obj.get_update_strategy()
    assert str(expected_value) == str(results["rollingUpdate"]["maxUnavailable"])
additional sleep time may required. 
http://magna002.ceph.redhat.com/ocsci-jenkins/openshift-clusters/j260bu1cmni33t1/j260bu1cmni33t1_20260110T125456/logs/failed_testcase_ocs_logs_1768073732322/test_update_strategy_config_change%5bcsi-cephfsplugin-2-2%5d_ocs_logs/j260bu1cmni33t1/ocs_must_gather/quay-io-rhceph-dev-ocs-must-gather-sha256-895d2702a767937955247fd26323a66696e284de860b85be40d658e63ab30c10/namespaces/openshift-storage/apps/daemonsets.yaml

        - hostPath:
            path: /var/lib/rook/openshift-storage.cephfs.csi.ceph.com/log/node-plugin
            type: DirectoryOrCreate
          name: csi-log
        - emptyDir: {}
          name: csi-logs-logrotate
    updateStrategy:
      rollingUpdate:
        maxSurge: 0
        maxUnavailable: 2
      type: RollingUpdate
http://magna002.ceph.redhat.com/ocsci-jenkins/openshift-clusters/j239vu1ce33ua/j239vu1ce33ua_20260112T225222/logs/failed_testcase_ocs_logs_1768324457635/test_update_strategy_config_change%5bcsi-cephfsplugin-2-2%5d_ocs_logs/j239vu1ce33ua/ocs_must_gather/quay-io-rhceph-dev-ocs-must-gather-sha256-895d2702a767937955247fd26323a66696e284de860b85be40d658e63ab30c10/namespaces/openshift-storage/oc_output/configmaps

Name:         rook-ceph-operator-config
Namespace:    openshift-storage
Labels:       <none>
Annotations:  <none>

Data
====
CSI_CEPHFS_PLUGIN_UPDATE_STRATEGY_MAX_UNAVAILABLE:
----
2

CSI_ENABLE_METADATA:
----
false

--- a/tests/functional/upgrade/test_configuration.py
+++ b/tests/functional/upgrade/test_configuration.py
@@ -1,4 +1,5 @@
 import logging
+import time
 
 import pytest
 
@@ -149,5 +150,6 @@ def test_update_strategy_config_change(
     ds_obj = DaemonSet(
         resource_name=daemonset, namespace=config.ENV_DATA["cluster_namespace"]
     )
+    time.sleep(10)
     results = ds_obj.get_update_strategy()
     assert str(expected_value) == str(results["rollingUpdate"]["maxUnavailable"])

